rts/lib/sol2/sol.hpp

```diff
@@ -9146,7 +9146,7 @@ namespace sol {
 
 	inline int push_type_panic_string(lua_State* L, int index, type expected, type actual, string_view message, string_view aux_message) noexcept {
 		const char* err = message.size() == 0
-		     ? (aux_message.size() == 0 ? "stack index %d, expected %s, received %s" : "stack index %d, expected %s, received %s: %s")
+		     ? (aux_message.size() == 0 ? "stack index %d, expected %s, received %s" : "stack index %d, expected %s, received %s: %s%s")
```

Comment:
could be a standalone fix

Reply:
this change to sol2 has been merged into upstream
we can just use the git submodule now instead of keeping a vendored copy